_posts/2018-09-19-announcing-riff-0-1-3-on-Knative.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: single
+title: "Announcing riff v0.1.3 on Knative"
+header:
+  overlay_image: /images/cam-bridge.jpg
+excerpt:
+  With a riff buildpack for java, credential helpers, content-type shortcuts, and subscription commands
+permalink: /blog/announcing-riff-0-1-3-on-knative/
+---
+
+We are pleased to announce the release of [riff v0.1.3](https://github.com/projectriff/riff/releases/tag/v0.1.3) on Knative. Thank you riff and Knative contributors.
+
+The riff CLI can be downloaded from our [releases page](https://github.com/projectriff/riff/releases/tag/v0.1.2) on GitHub. Please follow one of the [getting started](/docs) guides, to create a new cluster on GKE or minikube. This release includes new manifests for the latest Knative and Istio. 
+
+#### uninstall older Knative
+```sh
+riff system uninstall --istio --force
+```
+
+#### install on minikube (for GKE omit `--node-port`)
+```sh
+riff system install --manifest stable --node-port
+```
+
+## credential helpers
+Initializing a namespace with your push credentials is easier now. 
+
+#### point to your [GCR key](/docs/getting-started-with-knative-riff-on-gke/#create-a-kubernetes-secret-for-pushing-images-to-gcr)
+```sh
+riff namespace init default --gcr <path-to-json-file>
+```
+
+#### or `export DOCKER_ID=<your-id>` and enter password when prompted
+```sh
+riff namespace init default --dockerhub $DOCKER_ID
+```
+
+## riff buildpack for java
+This release of riff supports building Java functions from source, either locally or in the cloud. Both variants use a new riff buildpack for Java.
+
+All you need in your directory is the java code with a maven pom, and the name of the handler class in a file called 'riff.toml'. The example below uses a sample [hello-java](https://github.com/jldec/hello-java) function on GitHub.
```

Comment:
riff.toml should use backticks maybe?

Reply:
Should this read *available* on Github? The phrase "on github" seems to imply that you're directly building from there IMO (ie `--git-repo`)

Reply:
i tried backticks inline but this theme doesn't have good formatting for that and it just looks a bit off.

Reply:
i fiddled with the style for backticks inline. looks a bit better now.